apis/views.py

```diff
@@ -406,4 +417,66 @@ def get(self, request, election_id):
                             status=status.HTTP_200_OK)
         except NewElection.DoesNotExist:
             return Response({'detail': 'Get election detail failed', 'errors': {'detail': 'Election does not exist.'}},
-                            status=status.HTTP_404_NOT_FOUND)
\ No newline at end of file
+                            status=status.HTTP_404_NOT_FOUND)
+
+
+class ElectionVoteView(views.APIView):
```

Comment:
There are some missing information in Swagger docs.
![CleanShot 2565-11-04 at 18 11 25](https://user-images.githubusercontent.com/68165621/199959565-96d46a54-f6c3-4404-894f-fa5a56efb8cf.png)
- Parameters
- Short words